RNTester/e2e/__tests__/Button-test.js

```diff
@@ -25,18 +25,21 @@ describe('Button', () => {
   });
 
   it('Simple button should be tappable', async () => {
+    await element(by.id('example_search')).replaceText('Simple Button');
```

Comment:
This seems like a good helper

Reply:
Thanks for the suggestion, added helpers